.github/workflows/pinata-build-deploy-ui.yml

```diff
@@ -20,14 +21,13 @@ jobs:
 
       - name: Set environment for branch
         run: |
-          echo $GITHUB_BASE_REF
-          echo $GITHUB_REF
-          echo $GITHUB_HEAD_REF
           if [[ $GITHUB_BASE_REF == 'master' ]]; then
```

Comment:
I found an issue with this line. Looks like should be `refs/heads/master` not `master`.  See: https://github.com/Sifchain/sifnode/runs/1876970414?check_suite_focus=true#step:4:16